### PR TITLE
integrate fixed APR data to Markets page

### PR DESCRIPTION
**Pull request by ShawTim**

### CODE REVIEW

File: packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts [hunk 1]
```diff
@@ -274,4 +292,19 @@ describe('usePoolViewOptions', () => {
       }
     });
   });
+
+  it('useActivePoolList should return a list of active pools', () => {
+    const { result } = renderHook(() => useActivePoolList());
+    expect(result.current).toEqual(getConfigManager().getPoolList());
+  });
+
+  it('useInactivePoolList should return a list of active pools', () => {
```
Comment (lazarmitic) [inline packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts:301]: typo, text at the end should be `list of inactive pools`.

File: packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts [hunk 2]
```diff
@@ -274,4 +292,19 @@ describe('usePoolViewOptions', () => {
...
+    const { result } = renderHook(() => useInactivePoolList());
+    expect(result.current).toEqual([]);
+  });
+
+  it('useMaturedPoolList should return a list of active pools', () => {
```
Comment (lazarmitic) [inline packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts:306]: typo, text at the end should be `list of matured pools`.

**Review by lazarmitic — APPROVED** [review]

File: packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts (hunk 2 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts]: fixed

File: packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts (hunk 1 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/hooks/usePoolViewOptions.spec.ts]: fixed

**Review by iamvukasin — APPROVED** [review]

**Comment by danieledamiani** [comment]
in review

**Pull request merged**